docs/manual/help/index.md

```diff
@@ -1,22 +1,16 @@
-The first entry on Airtime's **Help** menu offers a **Getting Started** guide for new users. Further down, there is also a link to the online version of this **User Manual** and an **About** page, which displays version and licensing information.
+The first entry on Libretime's **Help** menu offers a **Getting Started** guide for new users. Further down, there is also a link to the online version of this **User Manual** and an **About** page, which displays version and licensing information.
 
 ![](static/Screenshot460-Help_page.png)
 
-You can visit the Airtime online support forum, and sign up for the mailing list, at <http://forum.sourcefabric.org/categories/airtime-support>
+You can visit the Libretime online support forum at <https://discourse.libretime.org/c/get-help> to ask for help from the community regarding your LibreTime setup. 
 
-This forum is mirrored by the mailing list, so posts on the forum appear on the mailing list and vice versa. You can therefore also post a message there by emailing <airtime-support@lists.sourcefabric.org>
-
-To subscribe to forum updates via email, please register or login to the forum by clicking the appropriate link. Then visit your profile page at <http://forum.sourcefabric.org/profile/> and click the link **My Email Subscriptions** on the left side menu. In the pop-up window that opens, check the boxes for the mailing lists you wish to subscribe to, then click the **Save** button.
 
 Bug reporting
 -------------
 
-Airtime needs your input to improve. If you think you've found a bug, please visit <http://dev.sourcefabric.org/> and sign in, using the same login and password that you registered for the Airtime forum. Create a bug report by selecting **Create Issue**, then **Airtime**, and then **Bug**. That way, the Airtime team can keep track of your problem and notify you when it has been fixed. You can also suggest improvements and new features for Airtime on that site.
+Libretime needs your input to improve. If you think you've found a bug, please visit <https://github.com/LibreTime/libretime> and sign in. Create a bug report by selecting **Issues**, then **New Issue**. That way, the Libretime team can keep track of your problem and notify you when it has been fixed. You can also suggest improvements and new features for Libretime on that site.
```

Comment:
The `and sign in` can probably be deleted

Reply:
Will be in next PR.